Sources/TuistKit/Services/ListService.swift

```diff
@@ -40,9 +36,32 @@ class ListService {
             return FileHandler.shared.currentPath
         }
     }
+
+    private func string(for templates: [PrintableTemplate],
+                        in format: ListService.OutputFormat) throws -> String {
+        switch format {
+        case .table:
+            let textTable = TextTable<PrintableTemplate> { [
+                TextTable.Column(title: "Name", value: $0.name),
+                TextTable.Column(title: "Description", value: $0.description),
+            ] }
+            return textTable.render(templates)
+
+        case .json:
+            let json = try templates.toJSON()
+            return json.toString(prettyPrint: true)
+        }
+    }
+}
+
+extension ListService {
```

Comment:
Nitpick: I'd move `OutputFormat` to the declaration of `ListService`. If the reason why we are doing it is improving the readability and structure of the code we can use `// MARK` annotations for that. If I'm not wrong, I think the compiler would be happier processing everything in one shot, rather than having to construct things through extensions separately.

Reply:
Sure, done 💪